src/app/zap-templates/common/ClustersHelper.js

```diff
@@ -649,9 +649,8 @@ Clusters.init = async function(context) {
 //
 // Helpers: All
 //
-function asBlocks(promise, options)
-{
-  return promise.then(data => templateUtil.collectBlocks(data, options, this))
+function asBlocks(promise, options) {
+  return promise.then(data => data ? templateUtil.collectBlocks(data, options, this) : '')
```

Comment:
Please undo this change, and fix the caller in `chip_cluster_command_arguments_with_structs_expanded` to pass in an empty array as needed.